src/tls13.c

```diff
@@ -5047,6 +5249,14 @@ int SendTls13ServerHello(WOLFSSL* ssl, byte extMsgType)
     if (extMsgType == server_hello)
         ssl->options.serverState = SERVER_HELLO_COMPLETE;
 
+#ifdef WOLFSSL_DTLS13
+    if (ssl->options.dtls)
+        return dtls13_handshake_send(ssl, output, sendSz, sendSz,
+                                     server_hello, 0);
+#endif /* WOLFSSL_DTLS13 */
+
+    ssl->buffers.outputBuffer.length += sendSz;
```

Comment:
When `dtls13_handshake_send` is called this doesn't get advanced. Is there a reason I'm not seeing?

Reply:
it will be advanced by `dtls13_send_one_fragment`